packages/manager/modules/pci/src/projects/project/notebooks/add/add.component.js

```diff
@@ -3,10 +3,18 @@ import template from './add.html';
 
 export default {
   bindings: {
+    goBack: '<',
+    goToNotebooks: '<',
```

Comment:
Both `goToNotebooks` and `goBack` do the same thing... Do you think we can remove one?

Reply:
done